Ports/zsh/package.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=zsh
+version=5.8
+workdir=zsh-zsh-${version}
+files="https://github.com/zsh-users/zsh/archive/refs/tags/zsh-${version}.zip zsh-${version}.zip 9fb194253336ae5871913a583e2bdb4db117bed80700539b2524cff606c8ff6f"
```

Comment:
```suggestion
version=5.8
files="https://sourceforge.net/projects/zsh/files/zsh/${version}/zsh-${version}.tar.xz zsh-${version}.tar.xz dcc4b54cc5565670a65581760261c163d720991f0d06486da61f8d839b52de27"
```

Can you try if this works for you?

Reply:
It worked, thanks ;)